README.md

```diff
@@ -1,6 +1,18 @@
 # terraform-ap-modules
 DSP Analysis Platform Terraform Modules. This public repository contains Terraform modules that are meant to be Broad-agnostic and can potentially be deployed by outside parties. The general pattern for their deployment inside the Broad is for them to be referenced from a deployment in the private [terraform-ap-deployments repository](https://github.com/broadinstitute/terraform-ap-deployments), along with any Broad-specific infrastructure.
 
+## Workflow for Atlantis-managed deployments
+When making changes to modules deployed with Atlantis in the [terraform-ap-deployments repository](https://github.com/broadinstitute/terraform-ap-deployments), the following workflow must be followed:
+1. Create a PR in this repo with changes to the module(s). If changing a module that is a sub-module of another module (such as `terra-env`), remember to point the module reference to the PR branch.
+2. Create a PR in terraform-ap-deployments with the module reference of the changing module pointing to the modules repo PR branch.
+3. [Run a plan](https://github.com/broadinstitute/terraform-ap-deployments#3-iterate-using-atlantis) in the ap-deployments PR. If changing multiple environments/projects it is recommended to initially reach a green plan/apply for one of them before changing the others.
+4. When the plan above succeeds, obtain reviews for both PRs.
+5. Apply your changes. Sometimes errors will appear during the apply that TF did not catch in the plan. After adressing any such errors, repeat 3 and 4.
+6. Once the apply is successful, update any sub-module references in the ap-modules PR branch to a version tag one minor version higher than the previous version of that module, then merge the ap-modules PR.
+7. Tag the merge commit in master with any new versions from the last step, and new versions for any modules pointing to those new versions, usually `terra-env`. For example, if the PR involves changes to two submodules of `terra-env`, there should be 3 tags on the merge commit, one for each of the two changed submodules and a new terra-env version.
```

Comment:
Also think an example PR here would be useful (you can use the one I just submitted)